pages/pipelines/branch_configuration.md.erb

```diff
@@ -58,12 +56,12 @@ When you define a value in the `branch` attribute of a step, only builds on the
 
```

Comment:
```suggestion
When combining positive and negative patterns, any positive pattern must match, and every negative pattern must not match.
```

Reply:
oops, that was supposed to be a suggestion first time around.

Reply:
Great, thank you!

Reply:
@plaindocs was this meant to replace the paraphraph? I think it's merged three paragraphs together, and I'm not sure it makes sense 😅

Reply:
yeah, I suspect I mangled it using the GH suggestion interface. 

Looks like we just need the new text, right?